source/includes/dev/_microservice_dev_guide.md

```diff
@@ -195,6 +200,88 @@ Running tests under Docker is similar to running the actual microservice. The ea
+
+Once reviewed by a peer, your changes will be merged and will be ready for deployment to one of the live environments.
+
+Currently, the RW API has 3 different environments:
+
+- `dev` for internal testing and development of new features. There are no guarantees of stability or data persistence. While it's not bared from public access, it's meant to be used only by developers working on the RW API code, for testing, debugging and experimentation.
+- `staging` is a more stable environment, meant to be used by both the RW API developers as well as other developers working on applications built using the RW API. It aims to be functionally stable, but occasional interruptions may occur if needed as part of a process, and code is sometimes in "release candidate" status, meaning it can have some issues. Data is often relied on by users of this API, so be mindful when performing destructive actions.
+- `production` is meant to be as stable as possible, as it's used by real users.
+
+Each microservice repository has a branch matching the name of each of these 3 environments, and changes will always go from a feature branch to `dev`, then to `staging`, and finally to `production`. To push your changes across the different environments, you should:
+
+- Create a PR from the source branch to the target branch (from `dev` to `staging`, or from `staging` to `production`)
+- Deploy the code to the respective environment (we'll see how in a moment)
+- Test it with actual calls to the API, to validate that no side effects were introduced.
+
+Depending on the scale of the changes you're doing, it's recommended to use [git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with [semantic versioning](https://semver.org/). Also be sure to update the `CHANGELOG.md` accordingly, and the `package.json` or equivalent files if they refer a version number. It's also best practice to announce the changes you're about to deploy before doing so, so that other developers of RW API applications can be on the lookout for regressions, and can quickly get in touch with you should any undesired behavior change be detected.
```

Comment:
Would take a firmer stance here and say that the process is to announce _at least_ 14 days in advance of the intended production deployment in the `general` channel in the WRI API Slack (and to contact Ethan Roday if you're not in that Slack workspace). In rare cases if a hotfix is needed to fix a breaking change in production, the 14-day lead time can be circumvented, but an announcement still must be made.